### PR TITLE
Update openfisca serve rst documentation

### PR DESCRIPTION
**Pull request by sandcha**
Connected to #593 

Brings #593 content to openfisca organisation
And adds minor changes: fixup (missing tick) + bump version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 21.3.7 []()
```
Comment (Anna-Livia) [inline CHANGELOG.md]: This needs to be updated :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
+- Update openfisca_serve [rst](http://openfisca.readthedocs.io/en/latest/openfisca_serve.html) documentation
+  * Make obvious that gunicorn parameters can be used in `openfisca serve` command and explicit parameters' format
```
Comment (Anna-Livia) [inline CHANGELOG.md]: I don't think `explicit` is a verb.  maybe `make xxx more explicit`or `explain` ?

File: doc/source/openfisca_serve.rst [hunk 3]
```diff
@@ -11,7 +11,13 @@ openfisca serve
 Additional arguments
 --------------------
 
-``openfisca serve`` uses ``gunicorn`` under the hood. In addition to the arguments listed above, you can use any ``gunicorn`` arguments when running ``openfisca serve`` (e.g. ``--reload``, ``--workers``, ``--timeout``).
+``openfisca serve`` uses ``gunicorn`` under the hood. In addition to the arguments listed above, you can use any ``gunicorn`` arguments when running ``openfisca serve`` (e.g. ``--reload``, ``--workers``, ``--timeout``, ``--bind``).
+See:
```
Comment (Anna-Livia) [inline doc/source/openfisca_serve.rst]: Could this all fit in one line ? e.g. `(see XXX for more information)`

File: CHANGELOG.md (hunk 2 above)
Comment (maukoquiroga) [reply CHANGELOG.md]: It is not, `make xxx more explicit` sounds good

File: doc/source/openfisca_serve.rst (hunk 3 above)
Comment (maukoquiroga) [reply doc/source/openfisca_serve.rst]: It don't see any impediment to this being merged as it is, I'd leave it to @sandcha to decide.

Comment (sandcha) [reply doc/source/openfisca_serve.rst]: I suggest do it in another PR as it's an improvement and not blocking to merge Joãn commits.  

> Besides, this is the list arguments that we tested on command line and in configuration file. 
If we replace this list with a link to [gunicorn documentation](http://docs.gunicorn.org), we should find a link to the right gunicorn revision and I didn't find it.

**Pull request merged**